code/modules/holiday/holidays.dm

```diff
@@ -421,11 +487,30 @@
 /datum/holiday/pride/getStationPrefix()
 	return pick("Pride", "Gay", "Bi", "Trans", "Lesbian", "Ace", "Aro", "Agender", pick("Enby", "Enbie"), "Pan", "Intersex", "Demi", "Poly", "Closeted", "Genderfluid")
 
+/datum/holiday/stonewall
+	name = "Stonewall Riots Anniversary"
+	begin_day = 28
+	begin_month = JUNE
+
+/datum/holiday/stonewall/greet() //Not gonna lie, I was fairly tempted to make this use the IC year instead of the IRL year, but I was worried that it would have caused too much confusion.
+	return "Today marks the [text2num(time2text(world.timeofday, "YYYY")) - 1969]\th anniversary of the riots at the Stonewall Inn!"
```

Comment:
Shouldn't this be in the year 2561, or whenever ss13 takes place in, rather than the time right now?

Reply:
two lines up 
" //Not gonna lie, I was fairly tempted to make this use the IC year instead of the IRL year, but I was worried that it would have caused too much confusion."

Reply:
Whoops I'm illiterate.
Eeeh, I think the IC year would make more sense still, while nice, I'd like to try and keep messages as IC as possible.

Reply:
It was tempting. Very tempting. Main reason I decided against it is because it likely would have resulted in confusion if it showed up as being 500 more years ago than it is IRL